From 2f22af7f5532cb8136c8cac03a91d008312426cb Mon Sep 17 00:00:00 2001
From: Tsuyoshi Ozawa <ozawa@apache.org>
Date: Tue, 24 Nov 2015 17:35:38 +0900
Subject: [PATCH 1157/2848] YARN-4387. Fix typo in FairScheduler log message.
 Contributed by Xin Wang.

(cherry picked from commit 28dfe721b86ccbaf2ddcfb7e709b226ac766803a)
(cherry picked from commit 9e458c3e9e5d6ae7850b3d56309157e2213a6603)

Change-Id: I50e83c4f55ba352dce1800e8d7bdaea5d80158b7
---
 .../scheduler/fair/FairScheduler.java              |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
index ab62300..b5d58d4 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
@@ -487,7 +487,7 @@ protected void warnOrKillContainer(RMContainer container) {
         // containers on the RMNode (see SchedulerNode.releaseContainer()).
         completedContainer(container, status, RMContainerEventType.KILL);
         LOG.info("Killing container" + container +
-            " (after waiting for premption for " +
+            " (after waiting for preemption for " +
             (getClock().getTime() - time) + "ms)");
       }
     } else {
-- 
1.7.9.5

